openlibrary/plugins/upstream/mybooks.py

```diff
@@ -184,12 +177,18 @@ class MyBooksTemplate:
     def __init__(self, username, key):
         self.username = username
         self.user = web.ctx.site.get('/people/%s' % self.username)
-        self.key = key
+        self.key = key.lower()
         self.readlog = ReadingLog(user=self.user)
         self.lists = self.readlog.lists
         self.counts = self.readlog.reading_log_counts
 
-    def render(self, page=1, sort='desc', list=None):
+    def render(
+        self, page=1, sort='desc', list=None, q="", doc_count: int = 0, ratings=None
```

Comment:
```suggestion
        self, page=1, sort='desc', list=None, q="", doc_count: int = 0
```

This appears to never be read from! So create it as a var inside the fn.

Reply:
Nope nevermind :P